apiserver/resources_mig_test.go

```diff
@@ -1,353 +0,0 @@
-// Copyright 2017 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
-
-package apiserver_test
-
-import (
```

Comment:
I am happy to remove this, but some test are interesting as we will want to rewrite them (with mock and stuff, but anyway)